frontend/src/metabase-types/store/dashboard.ts

```diff
@@ -98,7 +99,7 @@ export interface DashboardState {
   dashcards: Record<DashCardId, StoreDashcard>;
   dashcardData: DashCardDataMap;
 
-  parameterValues: Record<ParameterId, ParameterValueOrArray>;
+  parameterValues: ParameterValuesMap;
```

Comment:
`ParameterValuesMap` allows `null` values while the previous type does not (incorrect)

Reply:
There are 2 other places in the project that use `Record<ParameterId, ParameterValueOrArray>` - should we update them too?